source/_data/templates.yml

```diff
@@ -45,13 +45,13 @@
 #    - angular.js
 #    - frontend
 
-# - name: Embark Vortex Template
-#   description: A React Application showing what can be done with Vortex when using Embark.
-#   thumbnail: vortex.png
-#   install: embark new AppName --template Horyus/vortex-demo-embark
-#   link: https://github.com/Horyus/vortex-demo-embark/
-#   tags:
-#     - react
+- name: Embark x ΞVTX Template
+  description: An Embark template showcasing the features of ethvtx. ethvtx is an Ethereum-ready DApp Redux store. It will manage all the data and interactions between your user and the Ethereum Blockchain and is perfectly suited for React DApps.
+  thumbnail: vortex.png
+  install: embark new AppName --template Horyus/ethvtx_embark#embark-4.0.1
+  link: https://github.com/Horyus/ethvtx_embark/tree/embark-4.0.1
```

Comment:
I don't think we want to nail this down to `4.0.1`

People will think it'll be obsolete once `4.x` is out.